### PR TITLE
doc: optimize description about preload, init-store and others

### PR DESCRIPTION
**Pull request by aroundabout**
# Main change 
1. init-store.sh
image
2. Give the link of example.groovy in docker compose
image
3. fix the todo in #294 
After review, sync with en version

### CODE REVIEW

**Comment by aroundabout** [comment]
@imbajin

File: content/cn/docs/images/images-server/621swaggerui示例.png
Comment (imbajin) [inline content/cn/docs/images/images-server/621swaggerui示例.png:1]: also add the refer in https://hugegraph.apache.org/docs/clients/restful-api/
image

File: content/cn/docs/quickstart/hugegraph-hubble.md [hunk 1]
```diff
@@ -296,6 +299,9 @@ bin/start-hubble.sh -d
 展示顶点类型和边类型的顶点索引和边索引。
 
 #### 4.3	数据导入
+
+> **注意**：目前推荐使用 [hugegraph-loader](/cn/docs/quickstart/hugegraph-loader) 进行数据导入
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-hubble.md:303]:
```suggestion
> **注意**：目前推荐使用 [hugegraph-loader](/cn/docs/quickstart/hugegraph-loader) 进行正式数据导入, hubble 内置的导入用来做**测试**和**简单上手**
```

File: content/en/docs/quickstart/hugegraph-hubble.md [hunk 2]
```diff
@@ -295,6 +298,9 @@ Graph mode：
 Displays vertex and edge indices for vertex types and edge types.
 
 #### 4.3 Data Import
+
+> **Note**：currently, we recommend to use [hugegraph-loader](/en/docs/quickstart/hugegraph-loader) to load data
```
Comment (aroundabout) [inline content/en/docs/quickstart/hugegraph-hubble.md:302]:
```suggestion
> **Note**：currently, we recommend to use [hugegraph-loader](/en/docs/quickstart/hugegraph-loader) to import data formally. The built-in import of `hubble` is used for **testing** and **getting started**.

**Review by imbajin — APPROVED** [review]
lgtm

**Pull request merged**